### PR TITLE
update pd-recover docs

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Update pd-recover docs.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3281
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: pd-recover.md [hunk 1]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
-ansible -i inventory.ini tidb_servers -m shell -a 'cat {{deploy_dir}}/log/tidb*.log | grep "init cluster id" | head -10'
-```
+### 获取 Cluster ID
 
-```
-10.0.1.15 | CHANGED | rc=0 >>
-2019/10/14 19:23:04.688 client.go:161: [info] [pd] init cluster id 6747551640615446306
-……
-```
+一般在 PD、TiKV 或 TiDB 的日志中都可以获取 Cluster ID。你可以直接去服务器上查看日志以获取 Cluster id。
```
Comment (TomShawn) [inline pd-recover.md:39]:
```suggestion
一般在 PD、TiKV 或 TiDB 的日志中都可以获取 Cluster ID。你可以直接在服务器上查看日志以获取 Cluster ID。
```

File: pd-recover.md [hunk 2]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
-10.0.1.13 | CHANGED | rc=0 >>
-[2019/10/14 07:06:35.516 +00:00] [INFO] [node.rs:229] ["alloc store id 4 "]
 
-10.0.1.14 | CHANGED | rc=0 >>
-[2019/10/14 07:06:35.734 +00:00] [INFO] [node.rs:229] ["alloc store id 5 "]
+### 获取已分配 ID
 
-10.0.1.15 | CHANGED | rc=0 >>
-[2019/10/14 07:06:35.418 +00:00] [INFO] [node.rs:229] ["alloc store id 1 "]
+在指定已分配 ID 时，需指定一个比当前最大的已分配 ID 更大的值。可以从监控中获取已分配 ID，也可以直接去服务器上查看日志。
```
Comment (TomShawn) [inline pd-recover.md:90]:
```suggestion
在指定已分配 ID 时，需指定一个比当前最大的已分配 ID 更大的值。可以从监控中获取已分配 ID，也可以直接在服务器上查看日志。
```

File: pd-recover.md [hunk 3]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
 
-10.0.1.21 | CHANGED | rc=0 >>
-[2019/10/15 03:15:05.826 +00:00] [INFO] [node.rs:229] ["alloc store id 2001 "]
+#### 从监控中获取已分配 ID（推荐）
 
-10.0.1.20 | CHANGED | rc=0 >>
-[2019/10/15 03:15:05.987 +00:00] [INFO] [node.rs:229] ["alloc store id 2002 "]
-```
+要从监控中获取分配的ID，需要确保你所查看的监控指标是**上一任 PD Leader** 的指标。从 PD Dashboard 中的 **Current ID allocation** 面板中可以获取最大的已分配 ID。
```
Comment (TomShawn) [inline pd-recover.md:94]:
```suggestion
要从监控中获取已分配的 ID，需要确保你所查看的监控指标是**上一任 PD Leader** 的指标。可从 PD Dashboard 中 **Current ID allocation** 面板获取最大的已分配 ID。
```

File: pd-recover.md [hunk 4]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
 
-### 部署一套新的 PD 集群
+### 从 PD 日志获取已分配 ID
```
Comment (TomShawn) [inline pd-recover.md:96]:
```suggestion
#### 从 PD 日志获取已分配 ID
```

File: pd-recover.md [hunk 5]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
+
+要从 PD 日志中获取分配的ID，需要确保你所查看的日志是**上一任 PD Leader** 的日志。运行以下命令获取最大的已分配 ID：
```
Comment (TomShawn) [inline pd-recover.md:98]:
```suggestion
要从 PD 日志中获取分配的 ID，需要确保你所查看的日志是**上一任 PD Leader** 的日志。运行以下命令获取最大的已分配 ID：
```

File: pd-recover.md [hunk 6]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
 
-```
-ansible-playbook start.yml --tags=pd
-```
+你也可以在所有 PD server 中运行上述命令，找到最大的值。
 
-旧集群可以通过删除 `data.pd` 目录后，重新启动 PD 服务。
+### 部署一套新的 PD 集群
+
+部署新的 PD 集群之前，需要停止当前的 PD 集群，然后使用 `--data-dir` 指定并删除旧的数据目录。
```
Comment (TomShawn) [inline pd-recover.md:115]:
```suggestion
部署新的 PD 集群之前，需要停止当前的 PD 集群，然后删除旧的数据目录（用 `--data-dir` 指定）。
```

**Comment by TomShawn** [comment]
LGTM
@rleungx PTAL

**Comment by ran-huang** [comment]
@rleungx @nolouch PTAL~ Thanks!

File: pd-recover.md [hunk 7]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
+### 从源代码编译
 
-#### 从 PD 日志获取 [info] Cluster ID（推荐）
+* [Go](https://golang.org/)：PD Recover 使用了 Go 模块，请安装 Go v1.13 或更新版本。
+* 在 [PD](https://github.com/pingcap/pd) 根目录下，运行 `make pd-recover` 命令来编译源代码并生成 `bin/pd-recover`。
 
-使用以下命令，从 PD 日志中获取 [info] Cluster ID：
+> **注意：**
+>
+> 一般来说，用户不需要编译源代码，因为发布的二进制文件或 Docker 中已包含 PD Control 工具。开发者可以参考以上步骤来编译源代码。
```
Comment (rleungx) [inline pd-recover.md:21]:
```suggestion
> 一般来说，用户不需要编译源代码，因为发布的二进制文件或 Docker 中已包含 PD Recover 工具。开发者可以参考以上步骤来编译源代码。
```

File: pd-recover.md [hunk 8]
```diff
@@ -5,155 +5,126 @@ aliases: ['/docs-cn/dev/pd-recover/','/docs-cn/dev/reference/tools/pd-recover/']
...
 # PD Recover 使用文档
 
-PD Recover 是对 PD 进行灾难性恢复的工具，用于恢复无法正常启动或服务的 PD 集群。PD Recover 会随 tidb-ansible 一起下载，位于 resource/bin/pd-recover。
+PD Recover 是对 PD 进行灾难性恢复的工具，用于恢复无法正常启动或服务的 PD 集群。
 
-## 快速开始
+## 安装 PD Recover
 
-### 获取 Cluster ID
+要使用 PD Recover，你可以[从源代码编译](#从源代码编译)，也可以直接[下载 TiDB 安装包](#下载-tidb-安装包)。
```
Comment (rleungx) [inline pd-recover.md:12]: Should it be `(#下载-TiDB-安装包)`?

Comment (TomShawn) [reply pd-recover.md:12]: I am afraid not. `#下载-TiDB-安装包` does not follow the anchor format rule and markdownlint will report an error.

Comment (rleungx) [reply pd-recover.md:12]: got it.

**Review by rleungx — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]

**Review by disksing — APPROVED** [review]

**Pull request merged**